docusaurus/docs/reference/ha/bootstrapping.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_label: Bootstrapping
+sidebar_position: 10
+---
+
+# Bootstrapping A Cluster 
+
+To bring up a controller cluster, one starts with a single node. 
+
+## Controller Configuration
+
+### Certificates
+
+Each controller requires appropriate certificates. The certificates for clustered controllers 
+have more requirements than those for a standalone server. See the [Certificates Reference](./certificates.md)
+for more information. 
+
+### Config File
+
+The controller requires a `raft` section.
+
+```yaml
+raft:
+  dataDir: /path/to/data/dir
+```
+The `dataDir` will be used to store the following:
+
+* `ctrl-ha.db` - the ziti model bbolt database
+* `raft.db` - the raft bbolt database
+* `snapshots/` - a directory to store raft snapshots
+
+Controller use the control channel listener to communicate with each other. Unlike
+routers, they need to know how to reach each other, so an advertise address must
+be configured.
```

Comment:
> so an advertise address must be configured.

On an implementation pov, I don't understand this because it feels like a chicken-egg situation. As a reader, you just told me they need to know how to reach each other but then tell me to update the controller to advertise a particular address. It's not clear to me how this helps a joiner... Didn't I have to tell the joiner to join an existing cluster? This seems odd?

Reply:
If you tell a joiner to join an existing cluster, the rest of the cluster needs to know how to dial the joiner. This information is provided by controllers whenever they dial other controllers. 
It's also possible for a joiner to be given an incorrect, but working address. For example, the advertise address may be a global address, but the joiner may get an IP address or an internal name. The controller should decide how it generally referenced.